### PR TITLE
[style] CaseScreen hi-fis

### PR DESCRIPTION
**Pull request by ronniebeggs**
### Description

- Refactored `CaseScreen` to match Figma Hi-Fis during final hackathon.
- Standardized styling across related components/screens.

[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

### Screenshots

[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy!"

## How to review

[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

- Ensure that styling matches Figma hi-fis.
- Check that styling logic is consistent and correct across all related components/screens.

## Next steps

[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

- Push `EligibilityForm` to hi-fi.
- Create standardized global font styles.
- Create a project style guide that establishes design standards/best practices.

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

[Case Forms](https://github.com/calblueprint/impact-fund/pull/27)
[Case Summary](https://github.com/calblueprint/impact-fund/pull/22)
[Eligibility Form](https://github.com/calblueprint/impact-fund/pull/29)

CC: @ronniebeggs

### CODE REVIEW

File: src/Components/CaseSummaryCard/CaseSummaryCard.tsx [hunk 1]
```diff
@@ -23,18 +23,21 @@ export default function CaseSummaryCard(caseData: Case) {
...
+              {' '}
+              • {formatDate(caseData.date)}
             </Text>
-            <View style={styles.threeDots}>
-              <ThreeDots />
-            </View>
           </View>
+          <Text style={styles.blurbText}>{caseData.blurb}</Text>
+        </View>
+        <View style={styles.bottomContainer}>
```
Comment (stephaniewong2) [inline src/Components/CaseSummaryCard/CaseSummaryCard.tsx:38]: Let's make this sticky/stuck!

File: src/Components/FormsCard/styles.ts [hunk 2]
```diff
@@ -14,10 +14,12 @@ export default StyleSheet.create({
     borderColor: colors.darkGrey,
   },
   topContainer: {
+    height: 26 + 6,
```
Comment (stephaniewong2) [inline src/Components/FormsCard/styles.ts]: Let's use 32 LOL

**Review by stephaniewong2 — CHANGES_REQUESTED** [review]
YOU ARE A STYLING GOD! Just have two small comments!

File: src/Components/CaseSummaryCard/CaseSummaryCard.tsx (hunk 1 above)
Comment (ronniebeggs) [reply src/Components/CaseSummaryCard/CaseSummaryCard.tsx:38]: What are you referring to here? Are you talking about the "Learn more on case website" button on the `CaseSummaryScreen`?

Comment (stephaniewong2) [reply src/Components/CaseSummaryCard/CaseSummaryCard.tsx:38]: Yeah so sorry left comment on wrong file
